tests/appservice/test_api.py

```diff
@@ -76,15 +76,15 @@ async def get_json(
             headers: Mapping[Union[str, bytes], Sequence[Union[str, bytes]]],
         ) -> List[JsonDict]:
             # Ensure the access token is passed as a header.
-            if not headers or not headers.get("Authorization"):
```

Comment:
It seems like twisted handles bytes based strings just fine, as they seemingly internally use bytes regardless for headers. However, this mock seems to not handle both. It needs to explicitly use the `b` prefix to work.